README.md

```diff
@@ -1,15 +1,18 @@
 # fosdem2023-cni-unchained
+
```

Comment:
all changes in this file are not "seen" by the render - i.e. the same page is rendered with / without these changes. 

I don't think they're really required.

Reply:
Just for the sake of markdownlint and my ease of mind xD